recipes/opensartoolkit/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "opensartoolkit" %}
+{% set version = "0.12.5" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/opensartoolkit-{{ version }}.tar.gz
+  sha256: 867feb004333beb8245cbb6c548753bdc4d35545b99ee4496013a830442ea145
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - pytest-runner
+    - python >=3.6
+    - setuptools
+    - setuptools
+  run:
+    - python >=3.6
+    - descartes
+    - fiona
+    - gdal >=2
+    - geopandas >=0.8
+    - godale
+    - imageio
+    - jupyterlab
+    - matplotlib-base
+    - numpy
+    - pandas
+    - psycopg2-binary
+    - pyproj >=2.1
+    - python
+    - rasterio
+    - requests
+    - retrying
+    - rtree
+    - scipy
+    - shapely
+    - tqdm
```

Comment:
Where did you get these from?

I see this in the source setup.py:
```
    install_requires=[
        "pyproj",
        "descartes",
        "godale",
        "geopandas>=0.8",
        "jupyterlab",
        "psycopg2-binary",
        "rasterio",
        "requests",
        "scipy",
        "tqdm",
        "imageio",
        "rtree",
        "retrying",
    ],

```

Reply:
You are using an earlier release. Now I understand. Is it okay to try to target 0.12.15 instead?